charts/prometheus-blackbox-exporter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: Prometheus Blackbox Exporter
 name: prometheus-blackbox-exporter
-version: 8.6.1
+version: 8.7.1
```

Comment:
```suggestion
version: 8.7.0
```

Reply:
stupid action of my part, i had your commit